cocotb/bus.py

```diff
@@ -70,18 +70,30 @@ def __init__(self, entity, name, signals, optional_signals=[], bus_separator="_"
                 See the *signals* argument above for details.
             bus_separator (str, optional): Character(s) to use as separator between bus
                 name and signal name. Defaults to '_'.
+            case_insensitive (bool, optional): Perform case-insensitive match on signal names.
+                Defaults to True.
```

Comment:
```suggestion
                Defaults to False.
```
...once Kaleb's review is in.

Reply:
This should also get a ``.. versionadded:: 1.5``marker (https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded).